src/auth-service/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:12 as base
+FROM node:current as base
```

Comment:
Hi @BenjaminSsempala 
This can be disastrous!
We need to specify a specific version that you have test the changes in and sure it works.
Using current means that the Dockerfile will always use the latest node version which we are not sure our changes will work since these are released frequently.
I would advise changing this to a specific version preferable 18 given its the latest stable node version. Node 19 is the current version though has no long term support(LTS)

Reply:
@123MwanjeMike I've tested out with node 18 as you suggested, Everything seems to be working well.